app/(main)/customer/my-page/[id]/_components/MyPageContents/MyPageContents.tsx

```diff
@@ -56,15 +47,10 @@ const MyPageContents = () => {
     defaultValues: async () => await getMember(),
   });
 
-  const onSubmit: SubmitHandler<initialValuesType> = async () => {
-    if (address) setAddressError(false);
-    setClick(true);
-  };
-
   const changeProfile = async () => {
     const { nickName, phoneNumber } = watch();
 
-    await patchMember({
+    const res = await patchMember({
       req: {
         nickName: nickName,
         phoneNumber: phoneNumber,
```

Comment:
더 깔끔하게 할 수 있을 것 같아요..!

```suggestion
      req: {
        nickName,
        phoneNumber,
```

Reply:
조언 감사합니다!